sale_quotation_products/models/product_product.py

```diff
@@ -10,6 +10,12 @@
 class ProductProduct(models.Model):
     _inherit = "product.product"
 
+    qty = fields.Float(
+        'Quantity',
+        compute='_compute_qty',
+        inverse='_inverse_qty',
+    )
+
     @api.model
     def fields_view_get(self, view_id=None, view_type='form',
                         toolbar=False, submenu=False):
```

Comment:
This is a personal position: I am not really a fan about editing views from the fields_view_get method.
In this case I propose:

1. create a new editable view inherit from the original one but with primary mode one, there made the changes we need (add buttons, remove fields, set fields readonly, editable bottom, etc)
2. remove entirely the fields_view_get method
3. at view edit `Search Products` button and add to context `form_view_ref: 'module.my_new_editable_view'`